pkg/util/containers/filter.go

```diff
@@ -309,17 +311,17 @@ func NewAutodiscoveryFilter(filter FilterType) (*Filter, error) {
 		includeList = config.Datadog.GetStringSlice("container_include_logs")
 		excludeList = config.Datadog.GetStringSlice("container_exclude_logs")
 	}
-	return NewFilter(includeList, excludeList)
+	return NewFilter(ft, includeList, excludeList)
 }
 
 // IsExcluded returns a bool indicating if the container should be excluded
 // based on the filters in the containerFilter instance. Consider also using
 // IsExcludedByAnnotation in parallel with this check.
```

Comment:
this second sentence can be removed as a result of this change

Reply:
Good catch ! I'll remove it